include/NAS2D/Mixer/Mixer.h

```diff
@@ -32,7 +32,7 @@ class Mixer
 
 public:
 	Mixer() = default;
-	virtual ~Mixer() = 0;
+	virtual ~Mixer() = default;
```

Comment:
I'm curious, why set one to explicit `default`, and the other left unspecified. I suppose the result is the same. It just feels like there is a difference in intent.